contributing.md

```diff
@@ -107,7 +107,7 @@ Please follow this process; it's the best way to get your work included in the p
 
    ```bash
    # Switch to the develop branch
-   git checkout develop
+   git checkout -b develop origin/develop
```

Comment:
Yes, this shouldnt be changed. Both ways will work the same IF develop branch doesnt already exist. Modern git will auto set up a new branch for you.

And the file was changed from 644 to 755 == no-no